app/assets/stylesheets/card-grid/card-grid.scss

```diff
@@ -6,137 +6,155 @@
+        right: 20px;
         opacity: 1;
         position: absolute;
         z-index: 2;
+        
+
+        input[type=checkbox] {
+            + label {
+                &:before {
+                    $checkbox-size:35px;
+                    font-family: "Font Awesome 5 Free";
+                    font-weight:400;
+                    content: "\f004";
+                    width: $checkbox-size;
+                    height: $checkbox-size;
+                    font-size: calc(#{$checkbox-size}*.7);
+                    border-radius:  $checkbox-size;
+                    padding:2px;
+                    cursor: pointer;
+                    opacity: .4;
+                    position: absolute;
+                    color: $color-light;
+                    //fix for mobile color issues;
+                    color: transparent;  
+                    text-shadow: 0 0 0 $color-light;
+                    background-color: $color-highlight-mid-import;
+                    @media only screen and (max-width: 500px) {
+                        top: 0;
+                    }
+                }
+            }
 
-        label {
-            position: relative;
+                &:checked+label::before {
+                    opacity: 1;
+                }
+                
+                &:checked+label::after {
+                    content: "";
+                    
+                }
 
-            &:before {
-                cursor: pointer;
-                opacity: .4;
-                content: "\2665";
-                position: absolute;
-                color: white;
-                font-size: 35px;
-                background-color: $color-highlight-mid-import;
-                padding: 6px;
-                border-radius: 25px;
-                line-height: 24px;
-                -webkit-tap-highlight-color: transparent;
-                transition: .25s;
-            }
 
-        }
+            }
 
-        &_checkbox:checked+label::before {
-            opacity: 1;
-        }
 
-    }
 
-    &_card {
-        width: 100%;
-        text-align: center;
+        }
 
-        img {
+        &_card {
             width: 90%;
-            margin: 0;
-        }
-    }
+            text-align: center;
+            position: relative;
 
-    &_counter {
-        display: flex;
-        margin: 0 auto;
-        justify-content: space-between;
-        align-items: center;
-        text-align: center;
-        width: 90%;
-        background-color: $color-highlight-import;
-        border-radius: 5px;
+            img {
+                width: 100%;
+                margin: 0;
+            }
+        }
 
-        [class*="-btn"] {
-            font-weight: bolder;
+        &_counter {
             display: flex;
-            justify-content: center;
-        }
+            margin: 0 auto;
+            justify-content: space-between;
+            align-items: center;
+            text-align: center;
+            width: 100%;
```

Comment:
```suggestion
            width: 35%;
```

Reply:
I have adjusted the scope of the PR to better show what should be reviewed.  So this falls a little our of that scope, buuuuuuuuuut this is a great point, and I will definitely be implementing it when I pull the card grid/gallery views into the profile page.